packages/common/src/utils.ts

```diff
@@ -93,6 +96,33 @@ function parseGethParams(json: any) {
     throw new Error('nonzero terminal total difficulty is not supported')
   }
 
+  let customHardforks: HardforksDict | undefined = undefined;
+  if (config.blobSchedule !== undefined) {
+    customHardforks = {};
+    const blobGasPerBlob = 131072;
+    for (const [hfKey, hfSchedule] of Object.entries(config.blobSchedule)) {
+      const hfConfig = hardforksDict[hfKey];
+      if (hfConfig === undefined) {
+        throw new Error(`unknown hardfork=${hfKey} specified in blobSchedule`);
+      }
+      const { target, max, baseFeeUpdateFraction: blobGasPriceUpdateFraction } = hfSchedule as { target?: number, max?: number, baseFeeUpdateFraction?: undefined };
+      if (target === undefined || max === undefined) {
```

Comment:
Why is `blobGasPriceUpdateFraction` not mandatory?

Reply:
The "fallback" logic now works different from target/max and blobGasPriceUpdateFraction, for blobGasPriceUpdateFraction it will do the fallback, but for max/target it has to be explicitly defined.